ui-modules/blueprint-composer/app/components/util/model/entity.model.js

```diff
@@ -1138,6 +1156,19 @@ function setParametersFromJson(incomingModel) {
     this.touch();
 }
 
+/**
+ * Set initializers from JSON.
+ *
+ * @param {Object} model
+ * @param {Array} initializers
+ */
+function setInitializersFromJson(initializers) {
+    if (!Array.isArray(initializers)) {
```

Comment:
I'm not sure throwing en Error here is right.  Should we be more tolerant of an incorrect type or null value here?

Reply:
I followed the pattern handling errors in this model. Brooklyn initialisers is expected to be an array.